cpp/include/cugraph/algorithms.hpp

```diff
@@ -2007,6 +2007,10 @@ void triangle_count(raft::handle_t const& handle,
                     raft::device_span<edge_t> counts,
                     bool do_expensive_check = false);
 
+template <typename vertex_t, typename edge_t, bool multi_gpu>
+rmm::device_uvector<edge_t> edge_triangle_count(
```

Comment:
> How do you match the returned output vector elements to the edges in the graph?

I see what you mean.The reason why my tests are passing is because the edges are sorted by `src` so the corresponding count will be return however this is not a correct assumption. I will return an `edge_property_t` instead and I believe you suggested before.